packages/core/src/spec/lib/TwaManifestSpec.ts

```diff
@@ -72,6 +72,7 @@ describe('TwaManifest', () => {
       expect(twaManifest.generateShortcuts())
           .toBe('[[name:\'shortcut name\', short_name:\'short\',' +
             ' url:\'https://pwa-directory.com/launch\', icon:\'shortcut_0\']]');
+      expect(twaManifest.fallbackType).toBe('customtabs');
```

Comment:
Can we add a test as well to check that `webview` works?

Reply:
done